src/components/AssetHeader/AssetActions.tsx

```diff
@@ -1,17 +1,28 @@
 import { ArrowDownIcon, ArrowUpIcon } from '@chakra-ui/icons'
 import { Button, ButtonGroup, Skeleton } from '@chakra-ui/react'
+import { CAIP19 } from '@shapeshiftoss/caip'
 import { useModal } from 'context/ModalProvider/ModalProvider'
 import { useWallet, WalletActions } from 'context/WalletProvider/WalletProvider'
-import { useAsset } from 'pages/Assets/Asset'
+import { selectAssetByCAIP19 } from 'state/slices/assetsSlice/assetsSlice'
+import { selectMarketDataById } from 'state/slices/marketDataSlice/marketDataSlice'
+import { AccountSpecifier } from 'state/slices/portfolioSlice/portfolioSlice'
+import { useAppSelector } from 'state/store'
 
-export const AssetActions = ({ isLoaded }: { isLoaded: boolean }) => {
-  const { asset, marketData } = useAsset()
+type AssetActionProps = {
+  isLoaded: boolean
+  assetId: CAIP19
+  accountId?: AccountSpecifier
```

Comment:
```suggestion
```
are we using this optional prop any more?

Reply:
This is that new thing we added, and it won't always be provided.